### DISCUSSION TITLE
Maximum Brightness Levels Supported by the Software

### DISCUSSION THREAD

**Discussion by mrabhishek007** [niklasr22/BrightIntosh#89] [General]
What is the highest brightness level attained by this software—1000 nits or 1600 nits?
If possible kindly mention in the application description. Many people can find this information helpful.
Thanks in advance

**Comment by niklasr22** [comment]
The increased maximum display brightness is 1000 nits, but 1600 nits peak brightness are still possible when viewing HDR content.
I will add this to the description, thanks for pointing this out.

**Reply by whitespacecode** [reply]
A percentage indicator could be great to.. When you find the 'sweet' spot you can always set it to that setting. Now it's just a slider with no information
